content/en/continuous_integration/guides/add_custom_metrics.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Add Custom Metrics to your Tests
+kind: guide
+---
+
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+This guide will walk you through adding and using custom metrics for your tests.
+
+### Add the custom metric to your test
+First step is to add the custom metric to your test by using the programmatic API:
+
```

Comment:
Maybe it's redundant, but I think it may be worth adding a bit here that mentions users should first follow the CI test visibility instructions for their language/framework.

Reply:
good idea

Reply:
done in https://github.com/DataDog/documentation/pull/18831/commits/9825c5dc76e81ab464a5320dfa78e08f60e33b3d